src/pages/Home/index.tsx

```diff
@@ -12,6 +11,11 @@ import useApi from 'hooks/API';
 
 const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
   const { data, error, loading } = useApi(`${process.env.REACT_APP_BOBA_FETCH}`, '/boba_list');
+
+  const { data: toppingsList } = useApi(`${process.env.REACT_APP_BOBA_FETCH}`, '/toppings_option');
```

Comment:
refactor by creating something like
`const BACKEND_API = process.env.REACT_APP_BOBA_FETCH` at the top and pass into useAPI

Reply:
updated at 72649f73c89f41e75275f9309cd6fef3fd561f91